pkg/util/helper/policy.go

```diff
@@ -139,3 +140,10 @@ func SetReplicaDivisionPreferenceWeighted(placement *policyv1alpha1.Placement) {
 	}
 	placement.ReplicaScheduling.ReplicaDivisionPreference = policyv1alpha1.ReplicaDivisionPreferenceWeighted
 }
+
+// SetDefaultGracePeriodSeconds Set the default value of GracePeriodSeconds in ApplicationFailoverBehavior
+func SetDefaultGracePeriodSeconds(behavior *policyv1alpha1.ApplicationFailoverBehavior) {
```

Comment:
Can we directly use `kubebuilder` to set the default value in the API definition?

Reply:
This field only takes effect when purge == `Graciously`. I'm afraid `kubebuilder` cannot judge this condition.